webpack.config.js

```diff
@@ -12,9 +13,19 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.js?$/,
```

Comment:
Should this be `test: /\.js$/,`? (note the omitted question mark)

Reply:
Yes it should! Patched.